portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -14,14 +14,31 @@
 
 package com.google.sps.data;
 
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+
 /** Class to store user's website comments. */
 public final class Comment {
 
   private final String message;
   private final String userData;
+  private final float score;
 
-  public Comment (String message, String userData) {
+  public Comment(String message, String userData) throws IOException {
     this.message = message;
     this.userData = userData;
+    this.score = getMessageScore(message);
+  }
+
+  private float getMessageScore(String message) throws IOException {
```

Comment:
Add a short comment on what this function does and in which cases it throws IOException. This may be unclear for a user who doesn't read your implementation (and in general users should not have to read your entire implementation to understand what a function/class does).

Reply:
Consider documenting the conditions in which IOException is raised, as it's not obvious at all.

e.g.
@throws IOException if fails to establish connection to LanguageService.

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#throwstag

Reply:
Thank you for the resource, I added @throws for the exception.